lib/strava/webhooks/models/subscription.rb

```diff
@@ -4,6 +4,8 @@ module Strava
   module Webhooks
     module Models
       class Subscription < Hashie::Trash
+        include Strava::Models::Mixins::HttpResponse
```

Comment:
Can we have this automatically added to any response? Make a base class that includes `HttpResponse` that inherits from `Hashie::Trash` and make all models inherit that.